Framework/Algorithms/src/FindPeaks.cpp

```diff
@@ -403,6 +403,13 @@ void FindPeaks::findPeaksUsingMariscotti() {
   if (!(w % 2))
     ++w;
 
+  if (!m_dataWS->isRaggedWorkspace() && (m_dataWS->blocksize() <= static_cast<size_t>(w))) {
+    std::stringstream errss;
+    errss << "Block size of the workspace should be greater than:" << w;
+    g_log.warning(errss.str());
```

Comment:
It's not necessary to print a warning as well as throw an error with the same message - it prints this
```
Block size of the workspace should be greater than:601
Error in execution of algorithm FindPeaks:
Block size of the workspace should be greater than:601
```

Reply:
Thanks for noting that! Did the changes as requested.